.github/PULL_REQUEST_TEMPLATE

```diff
@@ -20,22 +12,13 @@ Add mentions of things that are not covered here and are planed to be done in se
 
 ## Contributor Checklist:
 
-* [ ] I ran `tox -e lint` to format my patch to meet the [Twisted Coding Standard](https://docs.twistedmatrix.com/en/stable/core/development/policy/coding-standard.html)
-* [ ] I have created a newsfragment in src/twisted/newsfragments/ (see: [News files](https://github.com/twisted/trac-wiki-archive/blob/trunk//ReviewProcess.mediawiki#News_files))
-* [ ] The title of the PR starts with the associated issue number (without the `#` character).
-* [ ] I have updated the automated tests and checked that all checks for the PR are green.
```

Comment:
This part of the template change doesn't seem to be in support of resolving the linked ticket but I don't particularly object to the changes (one version seems as good as the other to me).

Reply:
> I have removed the contribution checklist as I think they are just noise.
> We should use the check list for real tasks associated with a PR.

This was a suggestion 

With the previous checklist, we see this in the reporting, but at least for me is noise.

![image](https://user-images.githubusercontent.com/204609/180539939-d017ed79-e1a1-4149-b225-10238841cb0c.png)

--------

if a ticket has no newsfragment, the CI will fail.

This is why I think that we don't need the check list... is extra work to tick :)